### ISSUE TITLE
Failed to run action within the VSCode

### ISSUE THREAD

**Issue by unbreakabl3** [vmware/vrealize-developer-tools#122]
### Description

When clicked on `Run vRO Action` button, I got the following error:

```
# Running getVmWithTag.js
# An error occurred: Could not import exec package into vRO: Command 'mvn dependency:copy -Dartifact=com.vmware.pscoe.o11n:exec::package -DoutputDirectory="/Users/user/Library/Application Support/Code/User/globalStorage/vmware-pscoe.vrealize-developer-tools" -Dmdep.stripVersion=true ' exited with code 1
```

### Steps to Reproduce

1. Create Javascript action
2. Select the action and click on the upper right corner the `Run vRO Action` button

**Expected behaviour:** [What you expect to happen]

Action will run

**Actual behaviour:** [What actually happens]
Failed with:

```
# Running getVmWithTag.js
# An error occurred: Could not import exec package into vRO: Command 'mvn dependency:copy -Dartifact=com.vmware.pscoe.o11n:exec::package -DoutputDirectory="/Users/user/Library/Application Support/Code/User/globalStorage/vmware-pscoe.vrealize-developer-tools" -Dmdep.stripVersion=true ' exited with code 1
```

**Reproduces how often:** [What percentage of the time does it reproduce?]
100%

### Environment
Mac OS 13.3

#### Client

-   vRealize Developer Tools Version: 2.4.0
-   vRealize Build Tools Version: 2.29.1
-   Visual Studio Code Version: 1.75.1
-   OS Version: Mac OS 13.3

#### Server

-   vRealize Automation Version: n/a
-   vRealize Orchestrator Version: 8.11.1

### Failure Logs

```
# Running getVmWithTag.js
# An error occurred: Could not import exec package into vRO: Command 'mvn dependency:copy -Dartifact=com.vmware.pscoe.o11n:exec::package -DoutputDirectory="/Users/user/Library/Application Support/Code/User/globalStorage/vmware-pscoe.vrealize-developer-tools" -Dmdep.stripVersion=true ' exited with code 1
```

**Comment by unbreakabl3** [comment]
Hello,
is there any chance to fix it or provide a temporarely workaround?

**Comment by VenelinBakalov** [comment]
Closed with https://github.com/vmware/vrealize-developer-tools/pull/140
Release in v2.5.1

**Comment by VenelinBakalov** [comment]
Opened an issue in Build Tools for Aria repo as well since the fix needs to be implemented there: 
https://github.com/vmware/build-tools-for-vmware-aria/issues/427

**Comment by VenelinBakalov** [comment]
@unbreakabl3 the `exec` package is re-added and will be released with the next Build Tools version release (most likely 2.44.0):
https://github.com/vmware/build-tools-for-vmware-aria/pull/435

Note that this means the `exec` package will be released as well with a version of 2.44.0 meaning that in order to run the action you will need to adjust the `Default version` setting which by default is set to 2.38.1
<img width="1145" alt="image" src="https://github.com/user-attachments/assets/55f5562e-a0bc-4d2f-a9bf-34e5a134a44a">


I will open a subsequent bug since the action now executes but does not show execution logs, only ID and result.

<img width="577" alt="image" src="https://github.com/user-attachments/assets/6bcd83e2-350d-4455-8339-5ec2c05f703d">
